apps/pie-docs/src/assets/styles/base/_layout.scss

```diff
@@ -99,20 +100,24 @@
     .c-layout-item,
     .c-content-container {
         h2 {
-            @include f.font-size('heading-xl', $scale: 'narrow');
+            @include p.font-size(--dt-font-heading-xl-size--narrow);
+            @include p.line-height(--dt-font-heading-xl-line-height--narrow);
```

Comment:
Question around this. Would it make sense to update the `to-px` function or have a font-size function that calculates the `px` and then returns a font-size and `line-height` together (setting a line-height to a default value if not passed in)? I think this may save on having to remember to include the line-height call everytime we include the font-size call what do you think?

Reply:
yea, i agree and I already shared it in the engineering channel but i decided to keep it out of the scope of this ticket as its gonna require more work that i initially anticipated.